ompi/include/mpi.h.in

```diff
@@ -626,7 +635,7 @@ typedef MPI_Win_errhandler_function MPI_Win_errhandler_fn
  * Predefined attribute keyvals
  *
  * DO NOT CHANGE THE ORDER WITHOUT ALSO CHANGING THE ORDER IN
- * src/attribute/attribute_predefined.c and mpif.h.in.
+ * ompi/attribute/attribute_predefined.c and mpif.h.in.
```

Comment:
Should mpif.h.in be mpif.constants.h? I noticed mpif.h.in just includes more files and the constants are in mpif.constants.h.
I also noticed a comment in ompi/attribute/attribute_predefined.c line 129 that says change mpi.h that maybe should be mpi.h.in.

Reply:
> Should mpif.h.in be mpif.constants.h? I noticed mpif.h.in just includes more files and the constants are in mpif.constants.h. 

Good point; I'll update.

> I also noticed a comment in ompi/attribute/attribute_predefined.c line 129 that says change mpi.h that maybe should be mpi.h.in.

👍